index.js

```diff
@@ -18,6 +18,28 @@ class BirthdayReminder extends q.DesktopApp {
         const monthOfTheBirthday = this.config.monthOfTheBirthday; // this is the birthday month entered by the user
         const dayOfTheBirthday = this.config.dayOfTheBirthday; // this is the birthday day entered by the user
 
+        const currentTimestamp = new Date(now.getFullYear(), currentMonth, currentDay); // get the current date year/month/day
```

Comment:
Why do you use this? Why don't you use const currentTimestamp = new Date();
Also don't call it timestamp. What you declared is a date